main.py

```diff
@@ -79,11 +79,16 @@ def convert(self, value, param, ctx):
                    'scaling down a node. This should be a comma-separated key=value '
                    'list.')
 @click.option("--scale-label", default=None)
+@click.option('--instance-type-priorities', default='', type=MULTIMAP_PARAM,
+              help='This should be a comma-separated key=value list.' +
+                   'ASGs with instance types that have priorities closer to 0 will' +
```

Comment:
No need for concatenate operator in python; consecutive lines of strings are concatenated by the parser.